src/cljs/athens/views/pages/block_page.cljs

```diff
@@ -102,7 +98,8 @@
   (let [state (r/atom {:string/local    nil
                        :string/previous nil})]
     (fn [block parents editing-uid refs]
-      (let [{:block/keys [string children uid]} block]
+      (let [{:block/keys [string children uid]} block
+            block-uid                           (:block/uid block)]
```

Comment:
`block-uid` isn't needed, you have this value already in `uid` from line above.

Reply:
😅